psi4/driver/driver_cbs_helper.py

```diff
@@ -323,12 +354,13 @@ def scf_xtpl_karton_2(functionname: str, zLO: int, valueLO: float, zHI: int, val
 
     if type(valueLO) != type(valueHI):
         raise ValidationError(
-            "scf_xtpl_karton_2: Inputs must be of the same datatype! (%s, %s)" % (type(valueLO), type(valueHI)))
+            f"scf_xtpl_karton_2: Inputs must be of the same datatype! ({type(valueLO)}, {type(valueHI)})")
 
     if alpha is None:
         alpha = 6.30
 
-    beta_division = 1 / (math.exp(-1 * alpha) * (math.exp(math.sqrt(zHI)) - math.exp(math.sqrt(zLO))))
+    #beta_division = 1 / (math.exp(-1 * alpha) * (math.exp(math.sqrt(zHI)) - math.exp(math.sqrt(zLO))))
```

Comment:
Please either remove or add a comment explaining the change. (After Peter's comment, of course.)